nginx/conf.d/default.conf

```diff
@@ -6,6 +6,11 @@ server {
   proxy_set_header X-Real-IP $remote_addr;
   proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
+  # decline any non-texastribune host
+  if ($host !~* ^(.*\.texastribune\.org)$ ) {
+        return 444;
+    }
```

Comment:
**nitpick:** small thing, but looks like the indentation is a little off here

Reply:
thank you. fixed.